GOVERNANCE.md

```diff
@@ -0,0 +1,204 @@
+
+While this approach has served us well, as the Project grows and faces more
+legal and financial decisions and interacts with other institutions, we see a
+need for a more formal governance model. Moving forward The Project leadership
+will consist of a Random Variables Council. We view this governance model as
+the formalization of what we are already doing, rather than a change in
+direction.
+
+
+Community Architecture
+-----------------------
+
+* N Person steering council (Must be odd, will be decided in first election)
+* Core Contributors (of which Council members are also a part of)
+* General Contributors
+
+Anyone working with ArviZ has the responsibility to personally uphold
+the Code of Conduct. Core Contributors have the additional responsibility
+of _enforcing_ the Code of Conduct to maintain a safe community.
+
+
+Random Variables Council
+------------------------
+The Project will have a Steering Council that consists of Core Contributors
+who have produced contributions that are substantial in quality and quantity,
+and sustained over at least one year. The overall role of the Council is to
+ensure, taking input from the Community, the
+long-term well-being of the project, both technically and as a community.
+
+During the everyday project activities, council members participate in all
+discussions, code review and other project activities as peers with all other
+Contributors and the Community. In these everyday activities, Council Members
+do not have any special power or privilege through their membership on the
+Council. However, it is expected that because of the quality and quantity of
+their contributions and their expert knowledge of the Project Software and
+Services that Council Members will provide useful guidance, both technical and
+in terms of project direction, to potentially less experienced contributors.
+
+Council Members will have the responsibility of
+* Removing members, including Council Members, if they are in violation of the Code of Conduct
+* Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+* Make decisions about strategic collaborations with other organizations or individuals.
+* Make decisions about the overall scope, vision and direction of the project.
+
+Note that each individual council member does not have the power to unilaterally wield these responsibilities, but the council as a whole must jointly make these decisions. In other words, Council Members are first and foremost Core Contributors, but only when needed they can collectively make decisions for the health of the project.
+
+ArviZ will be holding its first election to determine its initial council in the coming weeks and this document will be updated.
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
```

Comment:
public as in github/discourse or public as in slack (all core contributors would then have access, but general contributors would not)

Reply:
Updated wording. Let me know if it sounds better